File: web-app/online-warehouse/src/app/write-off-act/dto/write-off-act-goods.dto.ts
import {GoodsDto} from "../../shared/goods/dto/goods.dto";
import {WriteOffTypeEnum} from "./enum/write-off-type.enum";

export class WriteOffActGoodsDto {
  goods: GoodsDto;
  writeOffType: WriteOffTypeEnum;
  amount: number;
}

File: core/src/main/java/by/itechart/writeoffact/dto/WriteOffActGoodsDto.java
package by.itechart.writeoffact.dto;

import by.itechart.common.dto.GoodsDto;
import by.itechart.writeoffact.enums.WriteOffType;
import lombok.Data;

@Data
public class WriteOffActGoodsDto {
    private GoodsDto goods;
    private WriteOffType writeOffType;
    private Integer amount;
}
